Sources/MAVSDK-Swift/Generated/Passthrough.swift

```diff
@@ -0,0 +1,679 @@
+        }
+
+        public static func == (lhs: CommandInt, rhs: CommandInt) -> Bool {
+            return lhs.targetSystemID == rhs.targetSystemID
+                && lhs.targetComponentID == rhs.targetComponentID
+                && lhs.command == rhs.command
+                && lhs.current == rhs.current
+                && lhs.autocontinue == rhs.autocontinue
+                && lhs.params == rhs.params
+        }
+    }
+
+    public struct ParamsInt: Equatable {
+        public let param1: Float
+        public let param2: Float
+        public let param3: Float
+        public let param4: Float
+        public let x: Int32
+        public let y: Int32
+        public let z: Float
+
+        
+
+        public init(param1: Float, param2: Float, param3: Float, param4: Float, x: Int32, y: Int32, z: Float) {
+            self.param1 = param1
+            self.param2 = param2
+            self.param3 = param3
+            self.param4 = param4
+            self.x = x
+            self.y = y
+            self.z = z
+        }
+
+        internal var rpcParamsInt: Mavsdk_Rpc_Passthrough_ParamsInt {
+            var rpcParamsInt = Mavsdk_Rpc_Passthrough_ParamsInt()
+            
+                
+            rpcParamsInt.param1 = param1
+                
+            
+            
+                
+            rpcParamsInt.param2 = param2
+                
+            
+            
+                
+            rpcParamsInt.param3 = param3
+                
+            
+            
+                
+            rpcParamsInt.param4 = param4
+                
+            
+            
+                
+            rpcParamsInt.x = x
+                
+            
+            
+                
+            rpcParamsInt.y = y
+                
+            
+            
+                
+            rpcParamsInt.z = z
+                
+            
+
+            return rpcParamsInt
+        }
+
+        internal static func translateFromRpc(_ rpcParamsInt: Mavsdk_Rpc_Passthrough_ParamsInt) -> ParamsInt {
+            return ParamsInt(param1: rpcParamsInt.param1, param2: rpcParamsInt.param2, param3: rpcParamsInt.param3, param4: rpcParamsInt.param4, x: rpcParamsInt.x, y: rpcParamsInt.y, z: rpcParamsInt.z)
+        }
+
+        public static func == (lhs: ParamsInt, rhs: ParamsInt) -> Bool {
+            return lhs.param1 == rhs.param1
+                && lhs.param2 == rhs.param2
+                && lhs.param3 == rhs.param3
+                && lhs.param4 == rhs.param4
+                && lhs.x == rhs.x
+                && lhs.y == rhs.y
+                && lhs.z == rhs.z
+        }
+    }
+
+    public struct CommandLong: Equatable {
+        public let targetSystemID: UInt32
+        public let targetComponentID: UInt32
+        public let command: UInt64
```

Comment:
Should command be UInt64?

Reply:
Hm, you're right. So, in the c++ SDK it is Uint16 and the other ones are UInt8, since GRPC doesn't have these values, I decided to use the double architecture. Just so I remember what are the values in the C++ SDK: So the conversion is:
MAVSDK  |  GRPC
UInt8        |   UInt32 
UInt16      |   UInt64